src/content/topics/home/getting-started/index.mdx

```diff
@@ -27,7 +27,9 @@ To start using feature management, first create a feature flag in LaunchDarkly.
 
 Next, you need to connect your application to LaunchDarkly. You can connect to LaunchDarkly using any of our 20+ SDKs.
 
-Then, in your application, use the SDK to make a function call any time you want to determine whether a particular user should receive a feature, and what variation of the feature they should receive. Within milliseconds, the SDK analyzes the rules that exist in the flag and determines the correct feature variation to show the user. LaunchDarkly records every user who encounters a flag, and how the SDK evaluated the flag's rules for that user, so you can analyze application data easily.
+Then, in your application, use the SDK to make a function call any time you want to determine whether a particular user should receive a feature, and what variation of the feature they should receive. The SDKs use local caching to make these variation checks as efficient as possible.
+
+The SDKs send analytics events to LaunchDarkly to record every user who encounters a flag and which variation they received, so you can analyze usage patterns and remove temporary feature flags after a feature has been fully released (100% of traffic is receiving the same variation). You have complete control over which data is sent to LaunchDarkly and can designate certain data as private if you want LaunchDarkly to use it for flag evaluation but not to store it. The analytics data that our SDKs collect also enables [Experimentation](/home/about-experimentation) to help you measure how changes in your software are impacting metrics that matter to your team and your business.
```

Comment:
```suggestion
The SDKs send analytics events to LaunchDarkly to record every user who encounters a flag and which variation they received. You control which data is sent to LaunchDarkly and can designate certain data as private if you want LaunchDarkly to use it for flag evaluation but not store it.
```
@EmilHorne Hey, thanks for this PR! I've condensed it a bit to help maintain focus of this topic while still including the information you added. In the future, you can make PRs to /launchdarkly/git-gatsby, our internal docs repo. /LaunchDarkly-Docs is for folks who aren't members of the LaunchDarkly Github team. 

Thanks again for the suggestion!